assets/app/scripts/controllers/overview.js

```diff
@@ -334,6 +355,11 @@ angular.module('openshiftConsole')
       }
     };
 
+    function isCustomHost(route) {
+      var defaultHost = route.serviceName + '-' + route.metadata.namespace;
```

Comment:
@pweil- is the generated route host really `$serviceName-$namespace.$suffix`? that's vulnerable to collisions, since hyphens can appear both in namespaces and service names